build.sh

```diff
@@ -234,6 +234,9 @@ elif [ "$PIPELINE" = 'update' ]; then
         "$BUILD_CONTEXT"
     docker tag "$GAME_IMAGE" "$GAME_IMAGE_LAYERED"
     date
+else
+    echo "Unknown PIPELINE=$PIPELINE"
```

Comment:
how about this

```
    echo "Invalid PIPELINE '$PIPELINE'"
```

Reply:
whichever, doesn't really matter.

Reply:
also, this condition should be moved to the validation section in https://github.com/startersclan/docker-sourceservers/pull/132/files#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R157

Reply:
yes